db_test.go

```diff
@@ -74,3 +76,24 @@ func TestDB_Close_Sync(t *testing.T) {
 	err = db.Sync()
 	assert.Nil(t, err)
 }
+
+func TestDB_Concurrent_Put(t *testing.T) {
+	options := DefaultOptions
+	db, err := Open(options)
+	assert.Nil(t, err)
+	defer destroyDB(db)
+
+	wg := sync.WaitGroup{}
+	wg.Add(10)
+	for i := 0; i < 10; i++ {
+		go func() {
+			defer wg.Done()
+			for i := 0; i < 10000; i++ {
+				key := utils.GetTestKey(rand.Int())
+				e := db.Put(key, utils.RandomValue(128))
+				assert.Nil(t, e)
+			}
+		}()
+	}
+	wg.Wait()
+}
```

Comment:
Add some assertions after the concurrent put is done.
like `Assert.equal(100, KeysNum)`

Reply:
ok